src/screens/Login/Login.tsx

```diff
@@ -0,0 +1,15 @@
+import { Text, View } from 'react-native';
```

Comment:
Library imports on top, a space and then rest imports

Reply:
Resolved